src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -53,7 +55,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                 {
                     // what do we do if there's no master?
                     TargetBranch = x.FirstOrDefault(b => b.Name == "master");
-                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head.FriendlyName);
+
+                    // 'repo.Head' will be null in unit tests.
+                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head?.FriendlyName);
```

Comment:
Oh, `GitServiceHelper` is static for convenience so we can easily unit test things. It returns a mocked instance from the service provider when called in unit tests, created [right here](https://github.com/github/VisualStudio/blob/master/src/UnitTests/Substitutes.cs#L119)